alacritty/res/glsl3/text.v.glsl

```diff
@@ -23,9 +23,9 @@ flat out vec4 bg;
 uniform vec2 cellDim;
 uniform vec4 projection;
 
-uniform int backgroundPass;
+uniform int renderingPass;
 
-#define WIDE_CHAR 1
+#define HAS_WIDE_CHAR 2
```

Comment:
I thought that it's a bit clearer, since it's not a bit flag, but rather a packed sum. So 3 and 2 will mean that they have wide char.

Reply:
I'm mostly confused by the `HAS_` prefix tbh. I'd expect boolean prefix to either be `IS_` or nothing at all.

Reply:
That's intentional since it's not about identity, but about having, since it's a flag and not a property. So it could have `WIDE_CHAR` and `COLOR`, and it is wide char when the flag is higher than 2. For color it's colored when it's `1` or `3`, so you can't really check for `IS`, it's more about `has`.

Reply:
`has_widechar` would be more of a function name in that scenario though. For bitflags you'd name the flag itself `WIDE_CHAR` and then check if `flags.contains(WIDE_CHAR)`.